### PR TITLE
docs: update PR template

### PR DESCRIPTION
**Pull request by Pradumnasaraf**
### Changes
- Update the PR template

### CODE REVIEW

**Comment by Panquesito7** [comment]
I think this was already done in #888. 😅

**Comment by Pradumnasaraf** [comment]
OOPS, my bad 
Don't close this, I have to change the PR template so i will do in this one

**Comment by Pradumnasaraf** [comment]
> I think this was already done in #888. sweat_smile

@Panquesito7 
OOPS, my bad 
Don't close this, I have to change the PR template so I will do in this one

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -5,6 +5,6 @@ resolves #`issueNumber`
 
 ## Checklist for profile addition
 
-- [ ] Name the PR `Add <your-username>`
-- [ ] You've filled all the required fields in [data.json](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/data.json) file
-- [ ] You've linked the PR to correct Issue
+- [ ] Name the PR `fix: Add <your-username>`.
+- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory. 
+- [ ] You've linked the PR to correct Issue.
```
Comment (Panquesito7) [inline .github/pull_request_template.md:10]:
```suggestion
- [ ] You've linked the PR to the correct issue.
```

Comment (Pradumnasaraf) [reply .github/pull_request_template.md]: Thanks for the suggestions :)

**Review by Panquesito7 — APPROVED** [review]

**Review by schmelto — APPROVED** [review]

**Pull request merged**